requirements-test.txt

```
@@ -7,7 +7,7 @@
 #
 # We need at least version 6 to correctly identify Amazon Linux 2023
 # as using the dnf package manager.
-ansible>=6,<7
+ansible>=6
```

I am suggesting this updated version constraint. The lower limit is changed because neither `ansible` v6 nor `ansible-core` 2.13 (it's dependency) are being maintained. Although `ansible` v7 is no longer maintained, the version of `ansible-core` it uses (2.14) will be maintained through 2024-05-20. I am then suggesting an upper bound so that we are not caught off-guard by breaking changes in a new major version (like when they had to yank 9.0.0). Please see the [release and maintenance schedule for Ansible](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html) for supporting information. 

> [!NOTE]
> The comment above will need some updating and I'm not sure how you would prefer to document the new version constraint.

```suggestion
ansible>=7,<10
```

---

I'd say we should change it to `>=8,<10` since `ansible` v7 is unmaintained.  Are you OK with that?

---

That's fine with me. I went with 7 as the lower bound to be conservative but I am not attached.

---

Please see commit d8b06f3.